x-pack/plugins/lens/public/visualizations/xy/xy_config_panel/index.tsx

```diff
@@ -72,12 +72,13 @@ const getDataBounds = function (
 ) {
   const groups: Partial<Record<string, { min: number; max: number }>> = {};
   axes.forEach((axis) => {
-    let min = Number.MAX_VALUE;
-    let max = Number.MIN_VALUE;
+    let min = Number.MAX_SAFE_INTEGER;
+    let max = -Number.MAX_SAFE_INTEGER;
```

Comment:
This helper function failed to account for mixed or purely negative domains. I believe they mistook `Number.MIN_VALUE` for `Number.NEGATIVE_INFINITY`, assuming `Number.MIN_VALUE` to be a negative value.

So the domain could only be from `[5e-324, 1.7976931348623157e+308]`, otherwise this would return an undefined domain.

Related to issue 6 description and had caused bugs in other places.